src/hooks/useStats/useStatsUtils.ts

```diff
@@ -41,6 +41,26 @@ export function getTrackData(trackSid: string, statsReports: StatsReport[]) {
   return allCurrentTracks.filter((t) => t.trackSid === trackSid);
 }
 
+// Returns an array that only contains tracks that correspond to a simulcast layer
+// that is currently in use. All "configured" but inactive layers will be filtered out.
+export function removeInactiveLayers(
+  previousStats: StatsReport[] | undefined,
+  stats: StatsReport[] | undefined,
+  trackSid: string
+) {
+  if (!stats || !previousStats) return null;
```

Comment:
I think this line can be removed since line 134 will return null first. Basically, this line with never be executed, but the TS compiler can't figure that out. 

If we take it out, then this function will never return null, so we can change line 138 back to `trackData[0]` without the `!`.